bigbluebutton-html5/imports/ui/components/actions-bar/container.jsx

```diff
@@ -8,16 +9,35 @@ import { shareScreen, unshareScreen, isVideoBroadcasting } from '../screenshare/
 
 const ActionsBarContainer = props => <ActionsBar {...props} />;
 
-export default withTracker(() => ({
-  isUserPresenter: Service.isUserPresenter(),
-  isUserModerator: Service.isUserModerator(),
-  handleExitVideo: () => VideoService.exitVideo(),
-  handleJoinVideo: () => VideoService.joinVideo(),
-  handleShareScreen: (onFail) => shareScreen(onFail),
-  handleUnshareScreen: () => unshareScreen(),
-  isVideoBroadcasting: isVideoBroadcasting(),
-  recordSettingsList: Service.recordSettingsList(),
-  toggleRecording: Service.toggleRecording,
-  screenSharingCheck: getFromUserSettings('enableScreensharing', Meteor.settings.public.kurento.enableScreensharing),
-  enableVideo: getFromUserSettings('enableVideo', Meteor.settings.public.kurento.enableVideo),
-}))(ActionsBarContainer);
+export default withTracker(({ }) => {
+  const togglePollMenu = () => {
+    const showPoll = Session.equals('isPollOpen', false) || !Session.get('isPollOpen');
+
+    const show = () => {
+      Session.set('isUserListOpen', true);
+      Session.set('isPollOpen', true);
+      Session.set('forcePollOpen', true);
+    };
+
+    const hide = () => Session.set('isPollOpen', false);
+
+    if (Session.equals('isChatOpen', true)) Session.set('isChatOpen', false);
```

Comment:
Are you sure you want this sideeffect?

We probably want to have something more robust and flexible regarding on what is "open" in the sidebar. When we merge #6105 and #6178 we will have 3 possible states: Chat, Breakouts and Polls, otherwise we will endup with one session variable for each state and start to create eadge caes like: what happens if all of them are "open"? Is that a possible state?

What do you think @antobinary?